plugins/messages/app/helpers/messages_helper.rb

```diff
@@ -5,7 +5,7 @@ def format_subject(message, length)
       body = ''
     else
       subject = message.subject
-      body = truncate(message.body, length: length - subject.length)
+      body = truncate(message.body.to_plain_text, :length => length - subject.length)
```

Comment:
please use the new style with `:` instead of `=>`

Reply:
I agree. However discussing this on every MR seems arduous. I suggest adding `Style/HashSyntax` to our rubocop config to enforce this and then run auto-correct on the whole codebase to have a distinct commit.

This may affect more though - I've created an issue for this: https://github.com/foodcoops/foodsoft/issues/1018